lib/concurrent-ruby/concurrent/timer_task.rb

```diff
@@ -242,6 +267,24 @@ def execution_interval=(value)
       end
     end
 
+    # @!attribute [rw] interval_type
+    # @return [Symbol] method to calculate the interval between executions, can be either
+    #   :fixed_rate or :fixed_delay; default to :fixed_delay.
+    def interval_type
+      synchronize { @interval_type }
+    end
+
+    # @!attribute [rw] interval_type
+    # @return [Symbol] method to calculate the interval between executions, can be either
+    #   :fixed_rate or :fixed_delay; default to :fixed_delay.
+    def interval_type=(value)
```

Comment:
Consider removing this as discussed.

Reply:
Agreed, it should be only set when building a new TimerTask and not changed after